index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <section class="body">
+      <header>
+         <a href="#" class="welcome">Hi there</a>
+         <a href="#"><img src="images/Union.png" class="navigation_bar" alt="Navigation bar"></a>
```

Comment:
- [ ] Well done making some changes 😉 but the hamburger button should have a `nav` tag.
    - The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices.